gdal/scripts/fix_typos.sh

```diff
@@ -50,7 +50,7 @@ if ! test -d fix_typos; then
     mkdir fix_typos
     (cd fix_typos
      git clone https://github.com/rouault/codespell
-     git checkout codespell/gdal_improvements
+     (cd codespell && git checkout gdal_improvements)
```

Comment:
all the code below must also occur in the codespell subdirectory

Reply:
Really? The very original version did this:

```
cd codespell
git checkout gdal_improvements
cd ..
```

Reply:
You're right. Forget my above comment